releasenotes.txt

```diff
@@ -15,3 +15,8 @@ release notes for cppcheck-2.11
     - `constVariablePointer`
 - More command-line parameters will now check if the given integer argument is actually valid. Several other internal string-to-integer conversions will not be error checked.
```

Comment:
Not your line, but `will not be error checked.` should probably be `will now be error checked.`
Or we could remove the second sentence, since this is not relevant to the end user.